packages/component/src/Middleware/AttachmentForScreenReader/TextAttachment.js

```diff
@@ -3,16 +3,21 @@ import { hooks } from 'botframework-webchat-api';
 import PropTypes from 'prop-types';
 import React from 'react';
 
+import FileAttachment from './FileAttachment';
 import useUniqueId from '../../hooks/internal/useUniqueId';
 
 const { useLocalizer } = hooks;
 
-const TextAttachment = ({ attachment: { content = '' } = {} }) => {
+const TextAttachment = (activity, attachment, { attachment: { content = '' } = {} }) => {
   const labelId = useUniqueId('webchat__id');
   const localize = useLocalizer();
 
   const label = localize('ATTACHMENT_TEXT', content);
 
+  if (!attachment.content) {
```

Comment:
Prefer to do this in the middleware, other than in `<TextAttachment>` component.

Reply:
I spent hours trying to get it to work. it stays here.